classes/submission/sectionEditor/SectionEditorSubmission.inc.php

```diff
@@ -707,16 +707,19 @@ function getHighlightClass() {
 							strtotime($reviewAssignment->getDateNotified()) : 0;
 						$dateConfirmed = $reviewAssignment->getDateConfirmed() ?
 							strtotime($reviewAssignment->getDateConfirmed()) : 0;
+						$dateDue = $reviewAssignment->getDateDue() ?
```

Comment:
@asmecher sorry I cannot: I am a bit dummy with git, sorry.
Can you cherry pick this only, as I asked in https://github.com/pkp/pkp-lib/issues/1606?